pkg/skel/skel.go

```diff
@@ -256,15 +268,15 @@ func (t *dispatcher) pluginMain(cmdAdd, cmdCheck, cmdDel func(_ *CmdArgs) error,
 			if err != nil {
 				return types.NewError(types.ErrDecodingFailure, err.Error(), "")
 			} else if gtet {
-				if err := t.checkVersionAndCall(cmdArgs, versionInfo, cmdCheck); err != nil {
+				if err := t.checkVersionAndCall(cmdArgs, versionInfo, funcs.Check); err != nil {
```

Comment:
This requires `func.Check` must not be nil (otherwise nil pointer reference happen). Should we add validate all func.{Add,Del,Check,...} in initiliazation (or support 'nil' to mention unsupported)